sogs/db.py

```diff
@@ -223,7 +223,35 @@ def add_new_tables(conn):
             conn.execute("CREATE INDEX user_request_nonces_expiry ON user_request_nonces(expiry)")
 
         added = True
-
+    if 'inbox' not in metadata.tables:
+        if engine.name == 'sqlite':
+            conn.execute(
+                """
+CREATE TABLE inbox (
+    id INTEGER PRIMARY KEY,
+    recipient INTEGER NOT NULL REFERENCES users(id) ON DELETE CASCADE,
+    sender INTEGER NOT NULL REFERENCES users(id) ON DELETE CASCADE,
+    body BLOB NOT NULL,
+    signature BLOB NOT NULL,
+    expiry FLOAT DEFAULT ((julianday('now') - 2440587.5 + 1.0)*86400.0) /* now + 24h */)
```

Comment:
24h seems way too short

Reply:
Perhaps 7 days?  (Also the postgres version below is still defaulting to 15 days).

Reply:
Actually, we should make this admin-tunable.  i.e. keep the default here at something like 7d, but then add a config option for how many days and then explicitly specify it when inserting.